### ISSUE TITLE
Add functionality for accessibility AriaLabel + AriaDescribeby

### ISSUE THREAD

**Issue by maxgertzen20** [viclafouch/mui-otp-input#44]
Currently, the way the inputs are rendered is not accessible. You can pass a single "aria-label" to all components, but it should have similar functionality to the placeholder where you can set for each input a different "aria-label" with index.

Also, please add "aria-describeby" + "aria-label" to the wrapping Box element to improve accessibility.

Something like this:
![image](https://github.com/viclafouch/mui-otp-input/assets/92924026/ccd56bfd-85f3-4f5b-94f9-220dd7a8151e)

**Comment by viclafouch** [comment]
Hello @maxgertzen 

Would love to help me by opening a PR on this ;)

**Comment by maxgertzen** [comment]
> Hello @maxgertzen
> 
> Would love to help me by opening a PR on this ;)

Done ;-)
https://github.com/viclafouch/mui-otp-input/pull/45

**Comment by Codensity30** [comment]
In the docs please provide an example of how to access the  OTP entered and verify it.
